accumulo/src/main/java/org/vertexium/accumulo/AccumuloGraph.java

```diff
@@ -57,6 +58,7 @@
 import static org.vertexium.util.IterableUtils.singleOrDefault;
 import static org.vertexium.util.IterableUtils.toList;
 import static org.vertexium.util.Preconditions.checkNotNull;
+import static org.vertexium.util.StreamUtils.stream;
 
 public class AccumuloGraph extends GraphBaseWithSearchIndex implements Traceable {
```

Comment:
I agree. Mainly because it looks like it is not going to be that simple (since the data is in a different table). I just wanted to bring it up so we are all in agreement extended data is not part of regular history.